packages/flutter/test/services/text_input_test.dart

```diff
@@ -1231,6 +1232,35 @@ void main() {
       expect(control.methodCalls, expectedMethodCalls);
     });
 
+    testWidgets('receives keyboardType changes', (WidgetTester tester) async {
```

Comment:
The test for this PR should probably be in `editable_text_test.dart`, so the tests in this file only test the `TextInput` directly, and not its interaction with `TextField`. Here is a similar test in `editable_text_test.dart` https://github.com/flutter/flutter/blob/6316b48bca8dc2929327d3471d466a78254945b6/packages/flutter/test/widgets/editable_text_test.dart#L3058-L3094

Reply:
Done.